libmamba/include/mamba/core/transaction.hpp

```diff
@@ -55,12 +42,8 @@ namespace mamba
         // Only use if the packages have been solved previously already.
         MTransaction(MPool& pool, const std::vector<PackageInfo>& packages, MultiPackageCache& caches);
 
-        ~MTransaction();
-
         MTransaction(const MTransaction&) = delete;
         MTransaction& operator=(const MTransaction&) = delete;
-        MTransaction(MTransaction&&) = delete;
-        MTransaction& operator=(MTransaction&&) = delete;
```

Comment:
This class should have a clear semantics:
- either everything can de defaulted, and we can remove all these methods.
- or we keep the declaration of the destructor and move semantics methods and we default them